src/App.css

```diff
@@ -218,9 +214,11 @@ body {
   margin-bottom: 20px;
 }
 
-.add-book form {
+.custom-form {
   display: flex;
   justify-content: space-between;
+  gap: 3rem;
+  width: 100%;
 }
 
```

Comment:
- [ ] Kindly add the profile logo at the left side of the page which is missing in your design 👍 

## Your design:
![image](https://github.com/abiy006/Bookstore-React/assets/50990925/14a04d58-7940-4079-9365-c05e958aaabe)

## Template:
![image](https://github.com/abiy006/Bookstore-React/assets/50990925/2e41f53f-1e31-412e-872a-f796d123fe14)